src/initializers/joi.ts

```diff
@@ -2,7 +2,7 @@ import * as _Joi from 'joi';
 import {isString} from 'lodash';
 import * as sanitizeHtml from 'sanitize-html';
 import {URL} from 'url';
-import { getLogger } from './log4js';
+import {getLogger} from './log4js';
```

Comment:
❓ Should those spaces be removed? Do you maybe have a different rule in linter because on node apps I remember keeping those spaces

Reply:
I applied prettier to the whole file